pkg/githubreconciler/internal/template/template.go

```diff
@@ -61,9 +61,10 @@ func (t *Template[T]) Embed(body string, data *T) (string, error) {
 	}
 
 	marker := t.identity + t.markerSuffix
+	note := fmt.Sprintf("\n\n> **Note:** If you need to make manual changes to this PR, apply the `skip:%s` label so the reconciler won't overwrite them.", t.identity)
```

Comment:
just to make sure I understand correctly, this update is for all the `changemanager`, `statusmanager` and `issuemanager` not only `changemanager` as mention in the PR title?

Reply:
Bah, good catch.  I only meant to change the `changemanager`, I switched to draft while I reconsider the approach.